src/js/components/Select/doc.js

```diff
@@ -204,6 +204,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: undefined,
   },
+  'select.options.box.hover': {
```

Comment:
Since the hover style applies only to a single option, I'm thinking we should use `select.option.hover` instead of `select.options.box.hover`. Thoughts?

Reply:
I think that would be cleaner for sure, and can make that change. I do have a couple questions about theming conventions for this component if you have a moment to discuss?

Reply:
I'm wondering if we should do something more general as well, so that hovering over suggestions in TextInput and MaskedInput also use the same styling. Do they already use `global.hover`?

Reply:
FYI, I investigated this further. Here is a summary of my findings:
- TextInput, MaskedInput, and Select are inconsistent in their implementations of hover styling.
  - TextInput - uses `global.hover` by applying the `hoverIndicator="background"` prop to its Buttons in the drop.
  - MaskedInput - is setting styling on the `background` prop of the option Button's child Box; it is currently using `colors.active`.
  - Select - is implemented similarly to MaskedInput, but slightly different.
- Calendar, Carousel, Menu, Video and other components also apply `global.hover` in the same manner as TextInput.
  - Tab, MaskedInput, and Select seem to be the outliers.

If we modify MaskedInput and Select, applying `hoverIndicator="background"`, we will get consistent hover styling across:
 - Calendar, Carousel, MaskedInput, Menu, Select, TextInput, Video


NOTE: `global.hover` styling is not applied when component is interacted with keyboard inputs (e.g. using arrowDown to progress through dropdown options on a TextInput w/ suggestions does not render the `global.hover` styling). I will log an issue for this.

**Recommendation**: Modify MaskedInput and Select to use the `hoverIndicator="background"` approach which will yield the greatest consistency of hover behavior across all components.